src/main/webapp/app/exercises/quiz/participate/quiz-participation.component.scss

```diff
@@ -158,43 +161,40 @@
         display: none;
     }
 }
+
 .show-mobile {
     @media (min-width: 700px) {
         display: none;
     }
 }
 
-.quiz-icons {
-    max-width: 40%;
+.autosave-text {
+    max-width: 30%;
+    text-align: center;
 }
 
-.stepwizardquiz {
-    display: flex;
-    flex-wrap: wrap;
-    max-height: 60px;
-    overflow: hidden;
-    position: relative;
-
-    @media (max-width: 767.98px) {
-        max-height: 45px;
+.quiz-navigation {
+    .draganddropcolor-question {
```

Comment:
Do we need to nest it in the quiz-navigation class? 
We are already defining `background-color: var(--quiz-participation-footer-dnd-background)` in line 22, which is also nested in a class. Can we simplify by reducing the nesting?

Reply:
Fixed